app/controllers/api/v1/customers/random_customers_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::Customers::RandomCustomersController < ApplicationController
+  def show
+    render json: Customer.order("RANDOM()").first
```

Comment:
This is too much logic in the controller. I suggest creating a method on `Customer` that does the logic instead.